src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
```

Comment:
```suggestion
import { MovieCard } from '../MovieCard';
```
данные лучше получать в App.js и передавать в компоненты.